### PR TITLE
Get correct mimetype on objectstores

### PR DESCRIPTION
**Pull request by rullzer**
Signed-off-by: Roeland Jago Douma <roeland@famdouma.nl>

### CODE REVIEW

**Review by icewind1991 — APPROVED** [review]

**Review by nickvergessen — APPROVED** [review]

File: lib/private/Files/ObjectStore/ObjectStoreStorage.php [hunk 1]
```diff
@@ -349,12 +349,7 @@ public function rename($source, $target) {
 
 	public function getMimeType($path) {
 		$path = $this->normalizePath($path);
-		$stat = $this->stat($path);
```
Comment (juliushaertl) [inline lib/private/Files/ObjectStore/ObjectStoreStorage.php]: Am I missing something or is in that case mime type detection only done by file extension? https://github.com/nextcloud/server/blob/5bf3d1bb384da56adbf205752be8f840aac3b0c5/lib/private/Files/Storage/Common.php#L247-L253

Same for the Local storage actually https://github.com/nextcloud/server/blob/5bf3d1bb384da56adbf205752be8f840aac3b0c5/lib/private/Files/Storage/Local.php#L52 Shouldn't that at least use the result from stat?

Comment (rullzer) [reply lib/private/Files/ObjectStore/ObjectStoreStorage.php]: For local it makes sense. For objectstore we have nothing better.
In anycase I woud do this for now. And create an issue for the rest.

Comment (juliushaertl) [reply lib/private/Files/ObjectStore/ObjectStoreStorage.php]: OK, thanks, yes I already thought that this would be all we have for object storage.

**Pull request merged**